wordpress-mobile/WordPress-iOS
# Use system colors for tooltip and snackbar components

Fixes #14136

This PR changes the colors used in all Notices (Tooltips and snackbar components), as listed below:

 os | iOS 13 or later | iOS 12 
--- | --- | --- | 
**Background** | inverted systemGray5 | rgba(44.0, 44.0, 46.0, 1.0) |
**Primary title** | inverted label | white | 
**Secondary title/text** | inverted secondaryLabel | rgba(235.0, 235.0, 245.0, 0.6) |
**Action** | primary blue (shade40) |  primary blue (shade40) |

#### Note: inverted colors are system colors of the opposite appearance. For example, inverted systemGray5 means that in light appearance, we are using the value of `systemGray5` from dark appearance, etc


here are some screenshots for reference

element | light appearance | dark appearance
--- | --- | --- | 
**FAB** | <img width="300" src="https://user-images.githubusercontent.com/34376330/88601695-e8287b80-d036-11ea-99e8-e3d9f48b2af1.png"> | <img width="300" src="https://user-images.githubusercontent.com/34376330/88601797-07270d80-d037-11ea-8c82-2c022ddd8d0b.png">
**QuickStart step** | <img width="300" src="https://user-images.githubusercontent.com/34376330/88601973-5ec57900-d037-11ea-9d0f-3425f79c29f5.png"> | <img width="300" src="https://user-images.githubusercontent.com/34376330/88601920-4bb2a900-d037-11ea-9503-196352b3dba7.png">
**QuickStart suggest** | <img width="300" src="https://user-images.githubusercontent.com/34376330/88602077-90d6db00-d037-11ea-9cc0-d6f5fe854f70.png"> | <img width="300" src="https://user-images.githubusercontent.com/34376330/88602098-9fbd8d80-d037-11ea-851e-effc570f70b2.png">
**Reader actions** | <img width="300" src="https://user-images.githubusercontent.com/34376330/88602188-ddbab180-d037-11ea-96bb-e95d92f79c27.png"> | <img width="300" src="https://user-images.githubusercontent.com/34376330/88602159-ced3ff00-d037-11ea-81f6-ed9b221331e0.png">


To test:
- make a fresh install (so that the FAB tooltip will be shown)
- create a site and tap "yes help me" when prompted, so that the quick start will begin
- perform actions in the reader
- in all these steps, verify that the tooltips, snackbars and toast look as expected and do not have visual bugs in both dark and light appearance, as well as on iOS 12.

PR submission checklist:
